Sources/ComposableArchitecture/Reducer.swift

```diff
@@ -75,6 +75,15 @@ public struct Reducer<State, Action, Environment> {
     }
   }
 
+  /// Combines a reducer with another given reducer by running each one on the state, and
+  /// concatenating both of their effects.
+  ///
+  /// - Parameter reducers: Another reducer.
+  /// - Returns: A single reducer.
+  public func combine(_ other: Reducer) -> Reducer {
```

Comment:
We can all workshop the name a bit, but what do you think of this?
```suggestion
  public func combine(with other: Reducer) -> Reducer {
```
I also don't know if we should be looking to the naming guidelines and call this `combining` (or `combined`?) 🤔

Reply:
We went for `combined(with:)` 👍